### PR TITLE
Ensures the grammar of the rules conforms to English language standards.

### PR DESCRIPTION
**Pull request by F-OS**
This mainly is here to ensure clarity around the rules when it comes to administration of the english language server, as far as I can see, none of my edits changed the idea or the letter of the rules, they simply modified the grammar, however I'll need confirmation on that from someone who can read the Russian rules, to ensure I didn't mess anything up.

### CODE REVIEW

File: contents/en/rules_en.md [hunk 1]
```diff
@@ -1,52 +1,52 @@
-|0. Administration applies these rules on its discretion.|
+|0. Administration applies these rules at their discretion.|
 | :------------ |
-|That means that Administration is free not to punish for violations if it's necessary, or if there's no grievances to the offender.|
+|That means that Administration is free not to punish for violations if it's necessary, or if there are no grievances to the offender.|
 
 |1.  Don't be an asshole.|
 | :------------ |
-|This is just a game, and we here for enjoyment. Long-timed conflicts, intentional harm to community, toxic behavior are not allowed, and will be punished. This rule doesn't regulate minor conflicts.|
+|This is just a game, and we here for enjoyment. Long-term conflicts, intentional harm to the community, and toxic behavior are not allowed and will be punished. This rule does not apply to minor conflicts.|
```
Comment (LudwigVonChesterfield) [inline contents/en/rules_en.md]: >implying this is a total grammar overhaul.
"And we here for enjoyment" sounds weird to me.

File: contents/en/rules_en.md [hunk 2]
```diff
@@ -1,52 +1,52 @@
...
+|This disrupts Administration's work and is punished harshly. If you want to change your account, you should discuss it with the Administration. Admins can use multi accounts for their purposes.|
 
 |4.  Intentional murder/isolation/infliction of grievous bodily harm without any reason.|
 | :------------ |
-|You should have strong cause for killing. It can be: selfdefence, long-time roleplayed conflict, the fulfillment of the order(The one who gave an order to you will be responsible) and the enforcement of the sentence.|
-|The violation always will be considerated in context of events, which were earlier. Usually, the one who generate the situation is the one who takes responsibility.|
-|Full isolation of the player = kill.|
-|Infliction of grievous bodily harm, which causes long healing in medbay (more than 30 minutes) punishes in half.|
+|You should have a strong case for killing. It can be self-defense, a conflict roleplayed over a long period of time, the fulfillment of a order (The one who gave an order to you will be responsible) or the enforcement of a sentence.|
```
Comment (LudwigVonChesterfield) [inline contents/en/rules_en.md]: I feel like it was supposed to be "cause" not "case". As in, "a strong reason for killing", not "a strong situation for killing".

Comment (LudwigVonChesterfield) [reply contents/en/rules_en.md]: Also, "the flfillment of an order", instead of "a order".

File: contents/en/rules_en.md [hunk 3]
```diff
@@ -1,52 +1,52 @@
...
+|Full isolation of a player is equivalent to killing them.|
+|Infliction of grievous bodily harm, which causes long healing in medbay (more than 30 minutes) is punished half as severely as murder.|
 
 |5. Do not use IC knowledge, which is unknown to your character.|
 | :------------ |
-|Do not use IC knowledge that you knew by ООС-way.|
-|Do not use other roles' difficult devices. Base operations, like wall dismantling, are free for all, but surgery and engine fitting could be done only by specialists. With the exception of related posts, like robotics, or Jacks-of-all-trades: scientists and RD.|
-|The crew is aware of existing of all special roles and have approximate information about their methods, but doesn't know anything about their skills and devices.|
+|Do not use knowledge gained from OOC sources in-game.|
+|Do not use other roles' difficult devices. Basic operations, like wall dismantling, are known by all jobs, but surgery and the engine should be limited to specialists. With the exception of related jobs, like robotics, or Jacks-of-all-trades: scientists and the RD.|
```
Comment (LudwigVonChesterfield) [inline contents/en/rules_en.md]: The ' serves no purpose here.

File: contents/en/rules_en.md [hunk 4]
```diff
@@ -1,52 +1,52 @@
...
-|Do not interrupt the setting and lore.
-|When you use flavor text, you don't allow to write character, story or something like this. Only your appearance (and some OOC notes if it's needed).
-|Do not take the famous people names (Adolf Hitler) and too much popular characters (Darth Wader). You can roleplaying not so famous heroes, but you should agree it with administration first.|
+|Your manner of speaking must fit to how you are roleplaying, but not to how you normally behave in internet chatrooms and the like. Ideally, your speech should try to fit within normal grammar rules and literary standards as much as possible. Without any emojis, internet-slang or text-cuts: your characters communicate via voice.|
+|Do not act in a way that disrupts the setting and lore.
+|When you use flavor text, do not use it to write information about your character, their story or anything of that sort. Only your appearance (and some OOC notes if it's needed).
+|Do not take names of famous figures (Adolf Hitler) or popular characters (Darth Wader). You can roleplay as less famous heroes, but you should discuss it with the administration first.|
```
Comment (LudwigVonChesterfield) [inline contents/en/rules_en.md]: Might be a sophistication on my part, but using "roleplay as less famous heros" implies that villains, anti-heroes, and etc-etc can't be roleplayed as. Use "figures"/ "characters" instead?

File: contents/en/rules_en.md [hunk 5]
```diff
@@ -1,52 +1,52 @@
...
 
 |7. Communicate correctly.|
 | :------------ |
-|It's hard to roleplaying, when adult characters, spaceship crew, talking like schoolboys. Please be correct as much as it possible: your text must be comfortable to read.|
-|There are two language-separated servers: Russian and English. General interaction must fit to the server's language.
+|It's hard to roleplay as adults crewing a spaceship when people talk like schoolboys. Please maintain correct spelling and grammar as much as possible: your text should be comfortable to read.|
```
Comment (LudwigVonChesterfield) [inline contents/en/rules_en.md]: The rule itself is slightly bizzare, as we have 6.1, where it is already said that your grammar should be proper.

File: contents/en/rules_en.md [hunk 6]
```diff
@@ -1,52 +1,52 @@
...
 
 |9. Out-of-game cooperation.
 | :------------ |
-|Any out-of-game ways of communication does not allowed. This is metagame, that will be punished hard.
+|Any out-of-game  communication is not allowed. This is metagaming and will be punished hard.
 
 |10. ERP.|
 | :------------ |
-|Erotic roleplay is restricted, because of autistic influence to the game process. Hugs'n'kisses are allowed, but no more then. You heard the man.
+|Erotic roleplay is restricted due to autistic influence to the game progress. Hugs'n'kisses are allowed, but nothing more than that. You heard the man.
```
Comment (LudwigVonChesterfield) [inline contents/en/rules_en.md]: Game's progress perhaps?

**Comment by F-OS** [comment]
@LudwigVonChesterfield fixed.

File: contents/en/rules_en.md (hunk 5 above)
Comment (F-OS) [reply contents/en/rules_en.md]: Hrm, good point, I don't want to delete or alter any rules severely without consulting with an admin first though.

**Comment by SpadesSlick** [comment]
Outdated.

**Pull request closed without merging**